src/api/graphql.ts

```diff
@@ -57,32 +32,24 @@ const handler = async (
     return
   }
 
-  const { parse, contextFactory, execute, schema } = (await envelop)({ req })
   const { operationName, variables, query } = parseRequest(req)
 
   try {
-    const response = await execute({
-      schema,
-      document: parse(query),
-      variableValues: variables,
-      contextValue: await contextFactory(),
-      operationName,
-    })
-
-    if (process.env.NODE_ENV !== 'production') {
-      if (Array.isArray(response.errors)) {
-        response.errors.forEach(console.error)
-      }
-
-      res.setHeader('cache-control', 'no-cache, no-store')
-    }
-
-    if (response.errors && isBadRequestError(response.errors[0])) {
-      res.status(400)
-    } else {
-      response.errors && res.status(500)
+    const response = await execute(
+      {
+        operationName,
+        variables,
+        query,
+      },
+      { req }
+    )
+
+    if (Array.isArray(response.errors)) {
+      // TODO: Return 400 on userError
+      res.status(500)
     }
 
+    res.setHeader('cache-control', 'no-cache, no-store')
```

Comment:
Should this header be added in production too?

Reply:
Yes, we are not caching graphql data for now